runtime/lua/vim/highlight.lua

```diff
@@ -36,12 +36,12 @@ end
 
 --- Highlight range between two positions
 ---
----@param bufnr number of buffer to apply highlighting to
----@param ns namespace to add highlight to
----@param higroup highlight group to use for highlighting
----@param start first position (tuple {line,col})
----@param finish second position (tuple {line,col})
----@param opts table with options:
+---@param bufnr number Buffer number to apply highlighting to
+---@param ns number Namespace to add highlight to
+---@param higroup string Highlight group to use for highlighting
+---@param start {line: number, col: number} Start position
+---@param finish {line: number, col: number} Finish position
```

Comment:
We should change all `number`s to `integer`.

Reply:
Fixed, and changed `number` to `integer` in other lua files too.